Gopkg.toml

```diff
@@ -77,3 +77,7 @@ required = [
 [prune]
   unused-packages = true
   go-tests = true
+
+[[constraint]]
+  name = "github.com/jpillora/go-ogle-analytics"
+  revision = "14b04e0594ef6a9fd943363b135656f0ec8c9d0e"
```

Comment:
Can we use a release or branch?

Reply:
There are no `tags` in this repository and using a branch such as `master` we might erroneously update the `vendor/.../go-gogle-analytics` accidentally without knowing.

To get this in I ran
```bash
$ dep ensure -add github.com/jpillora/go-ogle-analytics@14b04e0594ef6a9fd943363b135656f0ec8c9d0e
``` 
which was the most recent commit

Reply:
Can you talk with the maintainer? @vharsh